README.md

```diff
@@ -55,230 +57,82 @@ Log4j
 
-## Using feature flags metrics
+### Install the FF SDK Dependency
 
-Metrics API endpoint can be changed like this:
-
-```
-Config.builder()
-              .eventUrl("METRICS_API_EVENTS_URL")
-              .build();
-```
-
-Otherwise, the default metrics endpoint URL will be used.
-
-## Listen on events
-
-```
-client.on(Event.READY, result -> log.info("READY"));
-
-client.on(Event.CHANGED, result -> log.info("Flag changed {}", result));
-```
+The first step is to install the FF SDK as a dependency in your application using your application's dependency manager. You can use Maven, Gradle, SBT, etc. for your application.
 
-events will work even when streamEnabled is off.
+Refer to the [Harness Feature Flag Java Server SDK](https://mvnrepository.com/artifact/io.harness/ff-java-server-sdk) to identify the latest version for your build automation tool.
 
-## Connector
+This section lists dependencies for Maven and Gradle and uses the 1.1.4 version as an example:
 
-This is a new feature that allows you to create or use other connectors.
-Connector is just a proxy to your data. Currently supported connectors:
-* Harness
-* Local (used only in development)
+#### Maven
 
+Add the following Maven dependency in your project's pom.xml file:
+```pom
+<dependency>
+    <groupId>io.harness</groupId>
+    <artifactId>ff-java-server-sdk</artifactId>
+    <version>1.1.4</version>
+</dependency>
 ```
-LocalConnector connector = new LocalConnector("./local");
-client = new CfClient(connector);
-```
-
-## Storage
 
-For offline support and asynchronous startup of SDK use storage interface.
-When SDK is used without waitForInitialization method then it starts in async mode.
-So all flags are loaded from last saved configurations and it will use that values.
-If there is no flag on storage then it will be evaluated from defaultValue argument.
+#### Gradle
 
 ```
-final FileMapStore fileStore = new FileMapStore("Non-Freemium");
-LocalConnector connector = new LocalConnector("./local");
-client = new CfClient(connector, Config.builder().store(fileStore).build());
+implementation group: 'io.harness', name: 'ff-java-server-sdk', version: '1.1.4'
 ```
 
-## Update from controller or handler
+### A Simple Example
 
-this is useful only if webpush is used and in that case you need to disable
-stream
-```
-Config.builder()
-      .streamEnabled(false)
-      .build();
+After installing the SDK, enter the SDK keys that you created for your environment. The SDK keys authorize your application to connect to the FF client. All features of the Java SDK are provided by the base class called `CfClient`.
 
-cfClient.update(Message message)
+```java
+public class SimpleExample {
+    public static void main(String[] args) {
+        try {
+            /**
+             * Put the API Key here from your environment
+             * Initialize the CfClient
+             */
+            String apiKey = "<your api key here>";
```

Comment:
Could we define both the apiKey and the flag name here, and read from OS environment variables by default. i.e. something like:

```
# API Key
apiKey = os.getenv('FF_API_KEY', "changeme")

# Flag Name
flagName = os.getenv('FF_FLAG_NAME', "harnessappdemodarkmode")
```

It also might be worth including any import statements that are required at the top

Reply:
+1 on getting the apiKey from the ENV if it is set.

However, since they could use multiple flags, do we want to set an example that stores flagName in the environment? Wouldn't the common practice be to provide the flagName explicitly at each query to `cfClient`?

Reply:
Yes/no/maybe :-)   In my mind this is just a first snippet to get the customer running as quickly as possible with minimal changes.  
We talked to a few SEs about this approach, which they liked, so followed it here: https://github.com/harness/ff-dotnet-server-sdk#a-simple-example

For what its worth the harness platform uses a large enum that it iterates over to cal boolVariation for each one, so there are probably lots of ways to invoke a variation.   If we consider this a basic example to get the customer running code that connects to the harness platform, then I think we could provide more detailed examples and best practices in the examples folder and under further reading.